packages/trader/src/AppV2/Components/SymbolIconsMapper/symbol-icons-mapper.tsx

```diff
@@ -0,0 +1,173 @@
+        frxXAUUSD: MarketCommodityGoldusdIcon,
+        frxXPDUSD: MarketCommodityPalladiumusdIcon,
+        frxXPTUSD: MarketCommodityPlatinumusdIcon,
+        cryBTCUSD: MarketCryptocurrencyBtcusdIcon,
+        cryETHUSD: MarketCryptocurrencyEthusdIcon,
+        WLDEUR: MarketDerivedEurBasketIcon,
+        WLDGBP: MarketDerivedGbpBasketIcon,
+        WLDXAU: MarketDerivedGoldBasketIcon,
+        JD10: MarketDerivedJump10Icon,
+        JD100: MarketDerivedJump100Icon,
+        JD25: MarketDerivedJump25Icon,
+        JD50: MarketDerivedJump50Icon,
+        JD75: MarketDerivedJump75Icon,
+        stpRNG: MarketDerivedStepIcon,
+        WLDUSD: MarketDerivedUsdBasketIcon,
+        '1HZ10V': MarketDerivedVolatility101sIcon,
+        '1HZ100V': MarketDerivedVolatility1001sIcon,
+        '1HZ150V': MarketDerivedVolatility1501sIcon,
+        '1HZ250V': MarketDerivedVolatility2501sIcon,
+        '1HZ25V': MarketDerivedVolatility251sIcon,
+        '1HZ50V': MarketDerivedVolatility501sIcon,
+        '1HZ75V': MarketDerivedVolatility751sIcon,
+        frxEURAUD: MarketForexEuraudIcon,
+        frxEURCAD: MarketForexEurcadIcon,
+        frxEURCHF: MarketForexEurchfIcon,
+        frxEURGBP: MarketForexEurgbpIcon,
+        frxEURJPY: MarketForexEurjpyIcon,
+        frxEURNZD: MarketForexEurnzdIcon,
+        frxEURUSD: MarketForexEurusdIcon,
+        frxGBPAUD: MarketForexGbpaudIcon,
+        frxGBPCAD: MarketForexGbpcadIcon,
+        frxGBPCHF: MarketForexGbpchfIcon,
+        frxGBPJPY: MarketForexGbpjpyIcon,
+        frxGBPNOK: MarketForexGbpnokIcon,
+        frxGBPNZD: MarketForexGbpnzdIcon,
+        frxGBPUSD: MarketForexGbpusdIcon,
+        frxNZDJPY: MarketForexNzdjpnIcon,
+        frxNZDUSD: MarketForexNzdusdIcon,
+        frxUSDCAD: MarketForexUsdcadIcon,
+        frxUSDCHF: MarketForexUsdchfIcon,
+        frxUSDJPY: MarketForexUsdjpyIcon,
+        frxUSDMXN: MarketForexUsdmxnIcon,
+        frxUSDNOK: MarketForexUsdnokIcon,
+        frxUSDPLN: MarketForexUsdplnIcon,
+        frxUSDSEK: MarketForexUsdsekIcon,
+        OTC_AS51: MarketIndicesAustralia200Icon,
+        OTC_SX5E: MarketIndicesEuro50Icon,
+        OTC_FCHI: MarketIndicesFrance40Icon,
+        OTC_GDAXI: MarketIndicesGerman40Icon,
+        OTC_HSI: MarketIndicesHongKong50Icon,
+        OTC_N225: MarketIndicesJapan225Icon,
+        OTC_AEX: MarketIndicesNetherlands25Icon,
+        OTC_SSMI: MarketIndicesSwiss20Icon,
+        OTC_FTSE: MarketIndicesUk100Icon,
+        OTC_SPC: MarketIndicesUs500Icon,
+        OTC_NDX: MarketIndicesUsTech100Icon,
+        OTC_DJI: MarketIndicesWallStreet30Icon,
+        R_10: MarketDerivedVolatility10Icon,
+        R_100: MarketDerivedVolatility100Icon,
+        R_25: MarketDerivedVolatility25Icon,
+        R_50: MarketDerivedVolatility50Icon,
+        R_75: MarketDerivedVolatility75Icon,
+    };
+
+    const IconComponent = iconMap[symbol as keyof typeof iconMap];
+    return IconComponent ? <IconComponent iconSize={iconSize} /> : null;
```

Comment:
Please, ignore if irrelevant, but not so long ago BE was adding new symbol for which we have no icon. Might it be good idea, if instead of null we use some default icon? Like we do with accounts and currency

Reply:
i'll have to ask design first